### PR TITLE
[SDK] fix getMultiChainContractList

### PR DESCRIPTION
**Pull request by jnsdls**

### CODE REVIEW

File: packages/sdk/src/evm/core/sdk.ts [hunk 1]
```diff
@@ -549,21 +550,38 @@ export class ThirdwebSDK extends RPCConnectionHandler {
...
     const sdkMap: Record<number, ThirdwebSDK> = {};
 
     return contracts.map(({ address, chainId }) => {
+      if (!chainMap[chainId]) {
+        // if we don't have the chain in our list of supported chains then we can't resolve the contract type regardless, don't even try to set up the SDK
+        return {
+          address,
+          chainId,
+          contractType: async () => "custom" as const,
+          metadata: async () => ({}),
```
Comment (joaquim-verges) [inline packages/sdk/src/evm/core/sdk.ts:573]: what happens in the contract detail for this when i click on it?

**Review by joaquim-verges — APPROVED** [review]

File: packages/sdk/src/evm/core/sdk.ts (hunk 1 above)
Comment (jnsdls) [reply packages/sdk/src/evm/core/sdk.ts:573]: it'll ask you to configure the chain

**Pull request merged**